docs/archives.md

```diff
@@ -144,6 +155,83 @@ export default withArchive( posts, state => state.posts, 'blog' )( Blog );
 ```
 
 
+### Manual Pagination
+
+For manual pagination, use the `withPagedArchive` HOC instead of `withArchive`. This allows you to manually specify the page of the archive to load, and ignores Repress's internal page counter. Additionally, only the current page of items is passed to your component.
+
+This HOC passes the same props as `withArchive`, with the following differences:
```

Comment:
```suggestion
This Higher Order Component passes the same props as `withArchive`, with the following differences:
```

Reply:
HOC is already used consistently throughout the docs (and defined the first time it's used).